edwin/client/static/js/components/Timeline.js

```diff
@@ -82,6 +94,23 @@ Timeline.propTypes = {
   params: React.PropTypes.object.isRequired,
 };
 
+class ProgressIndicator extends React.Component {
+  render() {
+    let state = Array.from(this.props.runningTasks).join(', ');
```

Comment:
Huh. I didn't know about `Array.from` before this. Immutable sequences have a `.join` method as well though, so you probably could have just done `this.props.runningTasks.join(', ')`. This is fine as it is though.